core/embed/rust/src/lib.rs

```diff
@@ -39,6 +39,9 @@ pub mod translations;
 pub mod ui;
 pub mod strutil;
 
+#[cfg(feature = "debug")]
+pub mod debug;
```

Comment:
i think you need `#[macro_use]` here to be able to use dbg_println from outside, no?

Reply:
fixed, see abc2d798f6e08f0a443d261329420993526d09b0